themes/digital.gov/layouts/communities/single.html

```diff
@@ -10,7 +10,7 @@
           <div class="grid-col-12" data-edit-this="{{- $path -}}">
 
             <p class="breadcrumb">
-              <a href="{{ .Parent.Permalink }}" title="{{ .Parent.Title }}">
+              <a href="{{ .Parent.Permalink }}" title="{{ .Parent.Title }}" aria-label="Back to {{ .Parent.Title }}">
```

Comment:
I'm noticing this gets read twice in VoiceOver for a community page.


Ex: [Federal Communicators Network in CoP](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-arrow-alt-text/communities/federal-communicators-network/)

Reply:
Hmmm, I only see it once in the links list. Are you talking about first opening VoiceOver and it reads the community page title twice?

<img width="838" alt="Screen Shot 2022-10-13 at 9 30 41 AM" src="https://user-images.githubusercontent.com/104778659/195610681-4dd4fef4-8f82-4b38-a322-d0b23b6cd453.png">

Reply:
Not in links list. I just tabbed to the back link. That was getting read twice in VoiceOver on tabbing.

![image](https://user-images.githubusercontent.com/3385219/195614405-cd894d2a-6798-4a50-9fc8-5b8146c434e6.png)

Reply:
I've removed the title attribute to prevent the double read.